ui/lib/replication/addon/templates/index.hbs

```diff
@@ -1,19 +1,23 @@
-{{#if (eq model.mode 'unsupported')}}
-  <PageHeader as |p|>
-    <p.levelLeft>
-      <h1 class="title is-3 has-text-grey">
-        Replication unsupported
-      </h1>
-    </p.levelLeft>
-  </PageHeader>
-  <EmptyState
-    @title="The current cluster configuration does not support replication"
-  />
-{{else}}
-  <ReplicationSummary
-    @cluster={{model}}
-    @showModeSummary={{true}}
-    @onEnable={{action "onEnable"}}
-    @onDisable={{action "onDisable"}}
-  />
-{{/if}}
+<section class="section">
```

Comment:
just added section and div to file so it wouldn't span the whole width while waiting to load.